VideoCallKitQuickStart/ViewController.swift

```diff
@@ -309,11 +311,16 @@ extension ViewController : TVIRoomDelegate {
     
     func room(_ room: TVIRoom, didDisconnectWithError error: Error?) {
         logMessage(messageText: "Disconncted from room \(room.name), error = \(String(describing: error))")
-        
+
+        if !self.userInitiatedDisconnect, let uuid = room.uuid {
+            self.callKitProvider.reportCall(with: uuid, endedAt: nil, reason: .failed)
+        }
```

Comment:
There are some possible scenarios where a server initiated disconnect is not a call failure. For example, if the customer's application Server completes the Room. I know its not the function of this example, but should we check this error code anyways given that it is sample code?

I'll defer to @paynerc and @piyushtank on this one.

Reply:
Yes @ceaglest  I was wondering about the room completion scenario as well. Since its not an error scenario, we should not show `Call Failed Screen`. I would vote for checking error codes.

Reply:
We no longer fail the call if it was completed by the server.